docs/faq.md

```diff
@@ -157,3 +157,9 @@ You can read more about this transition [here](/posts/copr-srpms).
 ## A command failed in packit-service: how do I reproduce it locally?
 
 Please read [this document on how to reproduce locally](reproduce-locally).
+
+## Unlocking Efficiency: The Advantages of Using Packit for Seamless Service Integration
+
+- Packit uses various services Testing Farm, Copr, Koji, Bodhi…
+- These services can be used without Packit, but Packit intergrates their continuity (the build in Copr is created automatically when a new version is released).
+- The user does not have to wait for the results and just interprets them.
```

Comment:
here I would emphasize also that the results can be accessed from one place (though indirectly), the forge/dashboard